gslib/tests/test_json.py

```diff
@@ -0,0 +1,31 @@
+from gslib.tests import testcase
+from gslib.utils import json_six
+import six
+
+class TestJsonConversion(testcase.GsUtilUnitTestCase):
+  @classmethod
+  def setUpClass(cls):
+    cls.text_json = six.ensure_text('[{"integer": 1,"decimal": 3.14,"text": '
+                        '"testing, 1, 2"}]')
```

Comment:
W.r.t. alignment of args that you can't fit all on the same line, either use the "Aligned with opening delimiter" style, or the " 4-space hanging indent; nothing on first line" style.  If you go with a 4-space indent, don't put any args on the same line as the method name:

http://google.github.io/styleguide/pyguide.html#34-indentation

Reply:
Yep, you're right, formatting changed in that file.